modules/dr-restoring-cluster-state.adoc

```diff
@@ -0,0 +1,116 @@
+Restoring etcd member etcd-member-ip-10-0-143-125.ec2.internal from snapshot..
+2019-05-15 19:03:34.647589 I | pkg/netutil: resolving etcd-0.clustername.devcluster.openshift.com:2380 to 10.0.143.125:2380
+2019-05-15 19:03:34.883545 I | mvcc: restore compact to 361491
+2019-05-15 19:03:34.915679 I | etcdserver/membership: added member cbe982c74cbb42f [https://etcd-0.clustername.devcluster.openshift.com:2380] to cluster 807ae3bffc8d69ca
+Starting static pods..
+..starting kube-scheduler-pod.yaml
+..starting kube-controller-manager-pod.yaml
+..starting kube-apiserver-pod.yaml
+..starting etcd-member.yaml
+Starting kubelet..
+----
+
+.. Once the restore starts, run the script on your other master hosts.
+
+. Verify that the Machine Configs have been applied.
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command.
++
+----
+$ oc get machineconfigpool
+NAME     CONFIG                                             UPDATED   UPDATING
+master   rendered-master-50e7e00374e80b767fcc922bdfbc522b   True      False
+----
++
+When the snapshot has been applied, the `currentConfig` of the master will match the ID from when the etcd snapshot was taken. The `currentConfig` name for masters is in the format `rendered-master-<currentConfig>`.
+
+. Verify that all master hosts have started and joined the cluster.
+
+.. Access a master host and connect to the running etcd container.
++
+----
+[core@ip-10-0-143-125 ~] id=$(sudo crictl ps --name etcd-member | awk 'FNR==2{ print $1}') && sudo crictl exec -it $id /bin/sh
+----
+
+.. In the etcd container, export variables needed for connecting to etcd.
++
+----
+sh-4.2# export ETCDCTL_API=3 ETCDCTL_CACERT=/etc/ssl/etcd/ca.crt ETCDCTL_CERT=$(find /etc/ssl/ -name *peer*crt) ETCDCTL_KEY=$(find /etc/ssl/ -name *peer*key)
+----
++
+.. In the etcd container, execute `etcdctl member list` and verify that the three members show as started.
++
+----
+sh-4.2#  etcdctl member list -w table
+
++------------------+---------+------------------------------------------+------------------------------------------------------------------+---------------------------+
+|        ID        | STATUS  |                   NAME                   |                            PEER ADDRS                            |       CLIENT ADDRS        |
++------------------+---------+------------------------------------------+------------------------------------------------------------------+---------------------------+
+| 29e461db6be4eaaa | started | etcd-member-ip-10-0-164-170.ec2.internal | https://etcd-2.clustername.devcluster.openshift.com:2380 | https://10.0.164.170:2379 |
+|  cbe982c74cbb42f | started | etcd-member-ip-10-0-143-125.ec2.internal | https://etcd-0.clustername.openshift.com:2380 | https://10.0.143.125:2379 |
+| a752f80bcb0da3e8 | started |   etcd-member-ip-10-0-156-2.ec2.internal | https://etcd-1.clustername.devcluster.openshift.com:2380 |   https://10.0.156.2:2379 |
```

Comment:
https://etcd-0.clustername.openshift.com:2380 

should be 

https://etcd-0.clustername.devcluster.openshift.com:2380

Reply:
Thanks, updated.